Gin settings changes for 8.x-3.0-beta1

Adnan-cds opened the pull request:
Updating Git settings to reflect Gin 8.x-3.0-beta1 config schema.

[Latest schema](https://git.drupalcode.org/project/gin/-/blob/8.x-3.0-beta1/config/schema/gin.schema.yml)

Adnan-cds committed:
Fix: Gin settings changes

Updating Git settings to reflect Gin 8.x-3.0-beta1 config schema.

Adnan-cds commented:
Hi Finn,
So the latest round of Gin related test failures are cause by the [gin.settings.yml bundled](https://github.com/localgovdrupal/localgov/blob/2.x/config/install/gin.settings.yml) with the localgov profile.  It is incompatible with Gin 8.x-3.0-beta1.  I am updating it.

Adnan-cds commented:
Hi Finn,
So this is the [default config bundled](https://git.drupalcode.org/project/gin/-/blob/8.x-3.x/config/install/gin.settings.yml) with the Gin theme.  It [used to look like this](https://git.drupalcode.org/project/gin/-/blob/8.x-3.0-alpha37/config/install/gin.settings.yml).  I am not entirely sure what we should provide from the *localgov* profile now.  Since you added the [original localgov/config/install/gin.settings.yml](https://github.com/localgovdrupal/localgov/blob/2.x/config/install/gin.settings.yml), perhaps you can decide what to keep and what to drop? @finnlewis

finnlewis committed:
Add default Gin theme config for new settings like preset_accent_color.

finnlewis reviewed (approved):
Thanks @Adnan-cds - I've added a couple of other default settings, based on the default install, which seems to install just fine. If you're happy with that let's get this merged and released to unblock the pull requests on https://github.com/localgovdrupal/localgov_subsites/

Adnan-cds committed:
Fix: Gin settings changes.

Minor changes to conform with the latest config schema from Gin theme version
8.x-3.0-beta1.

Adnan-cds commented:
Hi Finn,
I have reapplied my [previous changes](https://github.com/localgovdrupal/localgov/commit/fa8b83cecb67f23c1f6ffaf3229db48a37c07b1a) on top of your [last change](https://github.com/localgovdrupal/localgov/pull/425/commits/b52eed2d2ec7b92600f9eabd68b72195c9ad20d6) so that gin.settings.yml conforms with the new [Gin config schema](https://git.drupalcode.org/project/gin/-/blob/8.x-3.0-beta1/config/schema/gin.schema.yml).  Now all the tests for the localgov profile pass in my local site.  Could you please take a quick look and confirm if I should merge this @finnlewis

finnlewis commented:
Thanks @Adnan-cds - looks good to me, sorry I seem to have inadvertently undone some of your changes. Let's get this merged and released so we can progress.

Adnan-cds commented:
Thanks Finn.  Merging...
